im_library/src/helpers/EditorMethods.ts

```diff
@@ -36,43 +36,43 @@ export function getTreeQueryIri(select: TTIriRef[]) {
 export function processComponentType(type: TTIriRef | undefined): any {
```

Comment:
can we do something like:
```
if(ComponentType.has(componentId)) return ComponentType[componentId];
else throw new Error(Invalid component type encountered while processing component types: componentId);
```
unless we want them here as a (sort of) list for visibility? however i think that can be accomplished by the vocab itself (to be able to view and add component shapes)